package.json

```diff
@@ -119,5 +119,8 @@
     "prettier": "node ./scripts/prettier/index.js write-changed",
     "prettier-all": "node ./scripts/prettier/index.js write",
     "version-check": "node ./scripts/tasks/version-check.js"
+  },
+  "dependencies": {
+    "semver": "^5.5.0"
```

Comment:
Maybe move to `devDependencies` with the others?

Reply:
Done 👍 

fwiw devDependencies on workspace top-level aren't really useful, since workspaces are typically always installed on development environment (we do however respect them, so if you run `yarn --production` from the root, we won't install them, which will make your scripts unusable).